geoapi/services/features.py

```diff
@@ -264,9 +270,40 @@ def fromRAPP(database_session, projectId: int, fileObj: IO, metadata: Dict, orig
         pathlib.Path(questionnaire_path).mkdir(parents=True, exist_ok=True)
         asset_path = os.path.join(questionnaire_path, 'questionnaire.rq')
 
+        # write questionnaire file (rq)
```

Comment:
This seems to be the only other reference to the `questionnaire file` string and here you specify the `rq` value. I think it is unclear from the base string (`questionnaire file`) that it is rq file and adding that would alleviate the need to specify it here, right?

Reply:
yes, i updated here to:
```
        # write questionnaire rq file
```
and above in docstr to:
```
        :param fileObj: questionnaire rq file

```